linode/manager
# upcoming: [M3-8245] – Update "Add Node Pools" description in LKE Create flow

## Description 📝
Conditionally update description for "Add Node Pools" section in LKE Create flow if LDE is enabled.

## Target release date 🗓️
6/24

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-06-13 at 12 13 06 PM](https://github.com/linode/manager/assets/114682940/91604a1a-c69f-4c4a-a9ca-5fafa2fafc29) | ![Screenshot 2024-06-13 at 12 12 57 PM](https://github.com/linode/manager/assets/114682940/10cfcdb3-87d1-4ffd-a52e-c0e6c9ff33d1) |

## How to test 🧪
### Prerequisites
1. Have the `linode_disk_encryption` tag on your account
2. Point at the alpha/dev environment
3. Have the Linode Disk Encryption (LDE) flag in our dev tool toggled on

### Verification steps
Without the tag and/or with the LDE flag in our dev tool toggled off, you should not see any LDE-related things in the UI. In this PR, that means you should see the same 

`Add groups of Linodes to your cluster. You can have a maximum of 100 Linodes per node pool.` 

copy as you do in prod. Otherwise, you should see 

`Add groups of Linodes to your cluster. You can have a maximum of 250 Linodes per node pool. Node Pool data is encrypted at rest.`

## As an Author I have considered 🤔
